test/cpp/ext/gsm/metadata_exchange_test.cc

```diff
@@ -151,12 +146,8 @@ TEST_F(MetadataExchangeTest, ClientAttemptDuration) {
 }
 
 TEST_F(MetadataExchangeTest, ServerCallDuration) {
-  Init(
-      /*metric_names=*/{grpc::internal::OTelServerCallDurationInstrumentName()},
-      /*resource=*/TestGkeResource(),
-      /*labels_injector=*/
-      std::make_unique<grpc::internal::ServiceMeshLabelsInjector>(
-          TestGkeResource().GetAttributes()));
+  Init(/*metric_names=*/{
+      grpc::internal::OTelClientAttemptStartedInstrumentName()});
```

Comment:
Shouldn't this be `OTelServerCallDurationInstrumentName()`?

Reply:
Fixed